### PR TITLE
Fix reverse content match

### PR DESCRIPTION
**Pull request by dorg-kanderson**
Add reverse_content_match boolean to http_check

### CODE REVIEW

**Comment by dorg-kanderson** [comment]
Please review and let me know if I constructed the spec correctly.

File: spec/classes/datadog_agent_integrations_http_check_spec.rb [hunk 1]
```diff
@@ -67,6 +68,7 @@
           threshold: 456,
           window: 789,
           content_match: 'foomatch',
+          reverse_content_match: false,
```
Comment (truthbk) [inline spec/classes/datadog_agent_integrations_http_check_spec.rb]: can you set this to `true`? `false` and `undef` are falsey, and thus would never render - which is what we're later checking when we do:
```
it { should contain_file(conf_file).with_content(%r{reverse_content_match: true}) }
```

File: spec/classes/datadog_agent_integrations_http_check_spec.rb [hunk 2]
```diff
@@ -90,6 +92,7 @@
         it { should contain_file(conf_file).with_content(%r{threshold: 456}) }
         it { should contain_file(conf_file).with_content(%r{window: 789}) }
         it { should contain_file(conf_file).with_content(%r{content_match: 'foomatch'}) }
+        it { should contain_file(conf_file).with_content(%r{reverse_content_match: false}) }
```
Comment (truthbk) [inline spec/classes/datadog_agent_integrations_http_check_spec.rb]: should be: `reverse_content_match: true`

Comment (dorg-kanderson) [reply spec/classes/datadog_agent_integrations_http_check_spec.rb]: Fixed

File: spec/classes/datadog_agent_integrations_http_check_spec.rb (hunk 1 above)
Comment (dorg-kanderson) [reply spec/classes/datadog_agent_integrations_http_check_spec.rb]: Fixed

File: templates/agent-conf.d/http_check.yaml.erb [hunk 3]
```diff
@@ -32,6 +32,9 @@ instances:
 <% if instance['window'] -%>
     window: <%= instance['window'] %>
 <% end -%>
+<% if instance['reverse_content_match'] -%>
+    reverse_content_match: '<%= instance["reverse_content_match"] %>'
```
Comment (truthbk) [inline templates/agent-conf.d/http_check.yaml.erb]: could you please remove the single quotes here around: `<%= instance["reverse_content_match"] %>`, we don't need those :)

Comment (dorg-kanderson) [reply templates/agent-conf.d/http_check.yaml.erb]: Good catch! Fixed!

**Comment by dorg-kanderson** [comment]
Please let me know if you need anything additional from me to get this merged.  Thank you!

**Pull request merged**